docs/advanced-configuration.md

```diff
@@ -399,15 +399,36 @@ does all the necessary metadata enrichment.
 
 OpenTelemetry Collector also has
 [native functionality for logs collection](https://github.com/open-telemetry/opentelemetry-log-collection).
-This chart soon will be migrated from fluentd to the OpenTelemetry logs collection.
 
-You already have an option to use OpenTelemetry logs collection instead of fluentd.
-The following configuration can be used to achieve that:
+**IMPORTANT:** This chart will be migrated from fluentd to the OpenTelemetry logs collection by default starting from
+version 0.86.0. Please ensure that "logsEngine" parameter is explicitly set to avoid unexpected changes during upgrade.
+
+The following configuration can be used to switch between fluentd and OpenTelemetry logs collection:
 
 ```yaml
-logsEngine: otel
+logsEngine: <fluentd|otel>
```

Comment:
> Will fluentd still be an acceptable configuration for the logsEngine? Is this yaml a suggestion or current specification for what users can use?

Yes, it still the default option.

> Why not logsEngine: <otel:fluentd>?

Because `|` represents `OR` better than `:` I think.